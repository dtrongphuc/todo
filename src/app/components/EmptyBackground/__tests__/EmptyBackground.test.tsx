import { shallow } from 'enzyme';
import EmptyBackground from '../EmptyBackground';

describe('App component', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<EmptyBackground />);
    expect(wrapper).toMatchSnapshot();
  });
});
