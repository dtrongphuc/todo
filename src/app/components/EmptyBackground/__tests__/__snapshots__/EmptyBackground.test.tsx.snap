// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches the snapshot 1`] = `
<Styled(styled.div)>
  <Styled(styled.div) />
</Styled(styled.div)>
`;
